### PR TITLE
Joker Suit/Coat

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
Was given sprites and multiple people told me to do it. Sprite credit goes to @severepwnage on the discord

## Why It's Good For The Game
We live in a society. Also it might be a cool alt clown outfit, for, you know, reasons.

## Changelog
:cl:
add: Added joker suit/coat
add: Joker outfit for admin spawning
imageadd: Added sprites for joker suit/coat
/:cl:

### CODE REVIEW

**Comment by severepwnage** [comment]
This is a certified hood classic

**Comment by severepwnage** [comment]
@qwertyquerty gave this a go also

**Comment by MarkSuckerberg** [comment]
hold up a minute, made a minor error

**Comment by Arrfur** [comment]
this represents out society...

File: code/modules/clothing/outfits/standard.dm [hunk 1]
```diff
@@ -425,9 +425,18 @@
 	mask = /obj/item/clothing/mask/breath
 	suit_store = /obj/item/tank/internals/oxygen
 
+/datum/outfit/debug
```
Comment (qwertyquerty) [inline code/modules/clothing/outfits/standard.dm]: /datum/outfit/debug ?

Comment (MarkSuckerberg) [reply code/modules/clothing/outfits/standard.dm]: whoops

**Comment by qwertyquerty** [comment]
Tested?

**Comment by MarkSuckerberg** [comment]
yep

**Pull request merged**